tests/e2e/utils/common.go

```diff
@@ -892,3 +895,238 @@ func CompareTwin(deviceTwin map[string]*MsgTwin, expectedDeviceTwin map[string]*
 	}
 	return true
 }
+
+// GetRuleList to get the rule list and verify whether the contents of the rule matches with what is expected
```

Comment:
Move those funcs to tests/e2e/utils/rule.go since some funcs do not be shared in all e2e.

Reply:
done